apps/organisations/forms.py

```diff
@@ -253,10 +257,42 @@ def __init__(self, project=None, format=None, *args, **kwargs):
                         'must be min. {} pixel wide and {} pixel tall. '
                         'Allowed file formats are png, jpeg, gif. The file '
                         'size should be max. 5 MB.'.
-                        format(sizes['img_width'], sizes['img_height']))
-
+                        format(self.sizes['img_min_width'],
+                               self.sizes['img_min_height'])),
         )
 
         if project:
             self.fields['title'].initial = project.name
             self.fields['description'].initial = project.description
+
+    def clean_image(self):
```

Comment:
This did not get in here from the a4 image stuff, right? I mean, it did not work with the sizes given here dynamically.

Reply:
This was the thing that I could not make work otherwise.. I tried to use the validate_image function from a4 images and just add that to the image fields validators, but that did not give back the size/height validation error, but 'image is corrupt', so I gave up on that and did it this way.

Reply:
Does that answer the question?